### DISCUSSION TITLE
Any plans for next release? Jan/Feb 2023?

### DISCUSSION THREAD

**Discussion by tunechr** [microsoft/presidio#1024] [General]
Hi All

There are a couple of fixes we are waiting on and was wondering if there was a plan to release a new version soon?

Cheers
Chris

**Comment by omri374** [comment]
Yes! this week most probably

**Reply by omri374** [reply]
https://github.com/microsoft/presidio/releases/tag/2.2.32

**Reply by tunechr** [reply]
Thanks for much Omri !
